distribution/docs/src/main/resources/_contents/_securing/environment-hardening-contents.adoc

```diff
@@ -40,11 +40,18 @@ On system: ensure that not everyone can change ACLs on your object.
 
 |SSH
 |tampering, information disclosure, and denial of service
-a|Create accounts for authentication (modify the default user's password) or remove ssh feature. By definition the connection will be secure when authenticated. Optionally, disable the ssh port (`default: 8101`) and set the `karaf.startRemoteShell` system property to false.
+a|By default, SSH access is disabled in ${branding}. To turn it on, edit the `etc/org.apache.karaf.shell.cfg`
+file, uncommenting the `sshPort` property. SSH can be also authenticated and authorized through an external Realm,
```

Comment:
❓ Should you add ddf.home to the path /etc/org.apache.karaf.shell.cfg?

Reply:
Good catch on the word order and branding home.